apps/els_lsp/src/els_completion_provider.erl

```diff
@@ -687,11 +687,17 @@ completion_item(#{kind := Kind = record, id := Name}, Data, _) ->
    , data             => Data
    };
 completion_item(#{kind := Kind = define, id := Name}, Data, _) ->
-  #{ label            => atom_to_binary(Name, utf8)
+  #{ label            => macro_label(Name)
```

Comment:
Im not sure how completions work. If there is no explicit `insertText` will this insert the label text upon selection? If yes, wouldn't it be better to not insert `/arity`? (Either insert just the name or an opening paren, or snippet with placeholder for args)

Reply:
I believe @gomoripeti 's interpretation is correct here.

Reply:
Ah, that makes sense. I wasn't really sure what the `label` does. I'll change it to only include the name in here.